Rtp/RtpClient.cs

```diff
@@ -3802,17 +3802,18 @@ public bool IListSockets
                     if (m_EventThread == null || EventsStarted.Equals(DateTime.MinValue))
                     {
                         //Create the event thread
-                        m_EventThread = new Thread(new ThreadStart(HandleEvents), Common.Extensions.Thread.ThreadExtensions.MinimumStackSize);
+                        m_EventThread = new Thread(new ThreadStart(HandleEvents), Common.Extensions.Thread.ThreadExtensions.MinimumStackSize)
```

Comment:
Lets stop passing `MinimumStackSize` it's the source of a lot of StackOverflow errors.